admin-manual.md

```diff
@@ -0,0 +1,187 @@
+# Running a MISO Instance
+MISO requires some configuration directly in the source code. While we plan to
+change this over time, running an instance of MISO will require building and
+deploying a fork of the code base with the customisations.
+
+# Creating a Fork and Setting Configuration
+Use the GitHub interface or a private instance to create a forked repository.
+
+Proceed to set up a build environment, then edit
+`miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes.
+The naming schemes will determine how MISO checks if object names (sample,
+library, etc) are valid. If you do not want to use one of the supplied ones
```

Comment:
'etc' in this context is not friendly, as it implies I have knowledge of MISO. Provide full list, or just give the two examples.